optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/SingleBenchmarkResult.java

```diff
@@ -55,8 +56,8 @@
     private Long usedMemoryAfterInputSolution = null;
 
     private Boolean succeeded = null;
-    private Boolean initialized = null;
     private Score score = null;
+    private Integer uninitializedVariableCount = null;
```

Comment:
uninitializedVariableCount should be before the field score, because it's logically earlier that. I am going to merge anyway already - if you could send in a separate PR later